<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="EstopOn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>EstopOn</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>168</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QPushButton" name="zadd">
       <property name="text">
        <string>z+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zless">
       <property name="text">
        <string>z-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>168</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="xadd">
       <property name="text">
        <string>x+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="xless">
       <property name="text">
        <string>x-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>168</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="yadd">
       <property name="text">
        <string>y+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="yless">
       <property name="text">
        <string>y-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>168</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QPushButton" name="Cadd">
       <property name="text">
        <string>c+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cless">
       <property name="text">
        <string>c-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>168</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="Aadd">
       <property name="text">
        <string>a+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Aless">
       <property name="text">
        <string>a-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>168</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QPushButton" name="Badd">
       <property name="text">
        <string>b+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Bless">
       <property name="text">
        <string>b-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="manual">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>manual</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mdi">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>mdi</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>168</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QPushButton" name="Eadd">
       <property name="text">
        <string>e+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Eless">
       <property name="text">
        <string>e-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>168</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QPushButton" name="Fadd">
       <property name="text">
        <string>f+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Fless">
       <property name="text">
        <string>f-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>168</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="Dadd">
       <property name="text">
        <string>d+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Dless">
       <property name="text">
        <string>d-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="EstopOff">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>EstopOff</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MachineOn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>MachineOn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MachineOff">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>MachineOff</string>
    </property>
   </widget>
   <widget class="QPushButton" name="automode">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>automode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="on">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>on</string>
    </property>
   </widget>
   <widget class="QPushButton" name="off">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="home">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>home</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testopencv">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>testopencv</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
